web/src/main/java/org/cbioportal/web/parameter/StudyViewFilter.java

```diff
@@ -36,6 +36,8 @@ public class StudyViewFilter implements Serializable {
     private List<List<String>> caseLists;
     private List<ClinicalDataFilter> customDataFilters;
     private AlterationFilter alterationFilter;
+
+    private List<DataFilter> clinicalEventFilters;
```

Comment:
I reused DataFilter class to store my EventType filter, but it may be more concise to create a class to hold the filter

Reply:
If this suits your case, I think DataFilter is good enough.